### ISSUE TITLE
parallelize wallet.refreshCounterpartyBalances

### ISSUE THREAD

**Issue by robby-d** [CounterpartyXCP/counterwallet#332]
Right now refreshCounterpartyBalances makes 3 API calls sequentially (it will be 4 once I add in my autobtcescrow stuff). we should use jquery's deferred/promises to make them all in parallel, and then call onSuccess after all return. This will speed up the balance syncs, since this function is commonly called and any lag is visible to the user (e.g. balances updating in parts, instead of all at once)
##
